### DISCUSSION TITLE
clickhouse插件使用问题

### DISCUSSION THREAD

**Discussion by XTUxiongda** [alibaba/ilogtail#1044] [Help] [answered]
我正在尝试借助ilogtail的clickhouse插件将数据采集到clickhouse中，我直接使用的官网的示例，配置如下
`enable: true
inputs:
  - Type: file_log
    LogPath: /logs/fofund-ipmc/
    FilePattern: "*.log"
processors:
  - Type: processor_json
    SourceKey: content
    KeepSource: false
    ExpandDepth: 1
    ExpandConnector: ""
flushers:
  - Type: flusher_stdout
    OnlyStdout: true
  - Type: flusher_clickhouse
    Addresses: 
      - 10.xxx.xxx.61:8123
    Authentication:
      PlainText:
        Database: default
    Table: demo`
但是ilogtail的控制台除了启动日志之外，别的日志一点都没有，就完全一点点反应都没有。
我只能通过调整我的配置文件，先确认了配置是生效的，可以正常在控制台输出，但是加上clickhouse这一段之后就又没反应了。
然后我就完全一点头绪都没有，不知道该如何去排查来调整我的配置或者环境，希望得到帮助，感谢

**Comment by kl7sn** [comment]
10.xxx.xxx.61:8123 修改为 10.xxx.xxx.61:9000

**Reply by cyenxchen** [reply, marked as answer]
当配置多个flusher插件的时候，只要其中一个flusher配置有问题就会导致flusher插件整体不生效，所以你这个问题应该是clickhouse的配置出问题了，你可以看下`logtail_plugin.LOG`日志文件。

**Reply by starriesWEB** [reply]
clickhouse只有buffer表有数据，log表没数据，咋回事呢
https://github.com/alibaba/ilogtail/issues/1202
